css/about.css

```diff
@@ -355,44 +377,45 @@ footer {
   margin: 1em 0.3em;
 }
 
-.container-footer-reserved {
-  display: flex;
-  flex-direction: row;
-  padding: 0 1.5em;
-  justify-content: space-between;
-}
-
 .footer-logo {
   display: flex;
-  justify-content: space-around;
   align-items: center;
   font-size: 1em;
-  width: 100%;
+  width: 50%;
 }
 
 .footer-logo p {
-  font-size: 1em;
+  font-size: 2em;
   font-weight: 900;
   line-height: 0.7em;
   margin-left: 4px;
 }
 
 .footer-logo .far {
-  font-size: 2em;
+  font-size: 3em;
 }
 
 .footer-paragraph-right-reserved {
-  font-size: 0.7em;
-  line-height: 1em;
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  margin-left: 0.8em;
-  margin-top: 5px;
+  font-size: 1em;
+  line-height: 1.5em;
 }
 
-.logo-word {
-  color: rgb(228, 228, 228);
+@media (max-width: 768px) {
+  .footer-right-reserved {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 3em 3em;
+  }
+
+  .footer-logo {
+    font-size: 0.4em;
+    width: 100%;
+  }
+
+  .footer-paragraph-right-reserved {
+    font-size: 0.8em;
```

Comment:
- [ ] [OPTIONAL] These are only marked as optional because the previous reviewer forgot to mention them but I strongly suggest you implement them.

- There is extra white space on the right of the homepage in the mobile view that causes horizontal scrolling and creates a bad user experience. Please get rid of the white space.
![image](https://user-images.githubusercontent.com/43995830/210766533-2fd52a3b-9992-48ef-8954-d975e15d7b9c.png)

- When the mobile menu is open, the program cards floats on top of it which is not supposed to be. Please kindly help fix this
![image](https://user-images.githubusercontent.com/43995830/210766860-3c41cc6a-836f-4cca-ba40-795fcdc815a8.png)